pyincore/utils/cgeoutputprocess.py

```diff
@@ -12,7 +12,7 @@ class CGEOutputProcess:
     """This class converts csv results outputs of Joplin CGE analysis to json format."""
 
     @staticmethod
-    def get_cge_household_count(household_count, household_count_path=None, filename_json=None):
+    def get_cge_household_count(household_count, income_categories, household_count_path=None, filename_json=None):
```

Comment:
Could you add the default `income_categories=["HH1", "HH2", "HH3", "HH4", "HH5"]` so it's backward compatible?

Reply:
("HH1", "HH2", "HH3", "HH4", "HH5")

Reply:
I agree with @longshuicy. Can you make this change backwards compatible? Then I think we can merge this.